src/index.html

```diff
@@ -9,7 +9,7 @@
 <body>
 <!-- Image and text -->
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <a class="navbar-brand" href="#">
+    <a class="navbar-brand" href="./index.html">
```

Comment:
we should really be using a click event to do this.  we don't want to refresh the page we want to show/hide the correct stuff.  Since this is outside of what the ticket asks lets return this back to what it was and write a bug ticket to fix it.

Reply:
Cool

Reply:
Done